In [2]:
import numpy as np 
import matplotlib.pyplot as plt

In [3]:
dataset = [32, 45, 28, 36, 53, 21, 17, 70, 65, 80, 68, 90, 75, 82, 5, 60, 55, 500]

In [4]:
sorted(dataset)

[5, 17, 21, 28, 32, 36, 45, 53, 55, 60, 65, 68, 70, 75, 80, 82, 90, 500]

In [5]:
quantile1,quantile3 = np.percentile(dataset,[25,75])

In [6]:
print(quantile1,quantile3)

33.0 73.75


In [7]:
iqr_value = quantile3 - quantile1

In [8]:
iqr_value

40.75

In [13]:
lower_bound = quantile1 - (1.5 *iqr_value)
upper_bound = quantile3 + (1.5 *iqr_value)

In [14]:
print(lower_bound, upper_bound)

-28.125 134.875


-28.125  to 134.875 is the range of deviation if any data is out of this range it's marked as outliers